documentation/modules/metrics/proc-metrics-kafka-deploy-options.adoc

```diff
@@ -5,45 +5,164 @@
 [id='proc-metrics-kafka-deploy-options-{context}']
 = Deploying Prometheus metrics configuration
 
-Strimzi provides xref:ref-metrics-yaml-files-{context}[example custom resource configuration YAML files] with relabeling rules.
+[role="_abstract"]
+Deploy Prometheus metrics configuration to use Prometheus with Strimzi.
 
-To apply metrics configuration of relabeling rules, do one of the following:
+You can use your own configuration or Strimzi's xref:ref-metrics-prometheus-metrics-config-{context}[example custom resource configuration files].
 
-* xref:proc-metrics-kafka-{context}[Copy the example configuration to your own custom resource definition]
-* xref:proc-metrics-deploying-kafka-{context}[Deploy the custom resource with the metrics configuration]
+* `kafka-metrics.yaml`
+* `kafka-connect-metrics.yaml`
+* `kafka-mirror-maker-2-metrics.yaml`
+* `kafka-bridge-metrics.yaml`
+* `kafka-cruise-control-metrics.yaml`
 
-[id='proc-metrics-kafka-{context}']
-== Copying Prometheus metrics configuration to a custom resource
+The example configuration files have relabeling rules and the configuration required to enable Prometheus metrics.
+Prometheus scrapes metrics from target HTTP endpoints.
+The example files are a good way to try Prometheus with Strimzi.
 
-To use Grafana dashboards for monitoring, copy xref:ref-metrics-yaml-files-{context}[the example metrics configuration to a custom resource].
+To apply the relabeling rules and metrics configuration, do one of the following:
 
-In this procedure, the `Kafka` resource is updated, but the procedure is the same for all components that support monitoring.
+* Copy the example configuration to your own custom resource definition
```

Comment:
`custom resource definition` can be easily confused with `CustomResoruceDefinition` a.k.a. CRD. So we should try to avoid it.

```suggestion
* Copy the example configuration to your own custom resources
```